modules/heat_conduction/test/src/materials/ADThermalConductivityTest.C

```diff
@@ -8,7 +8,7 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 #include "ADThermalConductivityTest.h"
 
-registerADMooseObject("HeatConductionApp", ADThermalConductivityTest);
+registerADMooseObject("HeatConductionTestApp", ADThermalConductivityTest);
```

Comment:
Nice catch. I was pretty puzzled why that documentation was required.

Reply:
Thanks to @gambka!